### PR TITLE
ICMSLST-788 Add setting to enable all workbasket rows & update script that creates test users.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/domains/case/models.py [hunk 1]
```diff
@@ -290,99 +291,121 @@ def get_workbasket_row(self, user: User) -> WorkbasketRow:
 
         task = self.get_active_task()
 
+        include_applicant_rows = (
+            settings.APP_ENV in ("local", "dev") and settings.SHOW_ALL_WORKBASKET_ROWS
```
Comment (oskusalerma) [inline web/domains/case/models.py:295]: non disastrous listing some links, lets just check the variable

File: config/settings/base.py [hunk 2]
```diff
@@ -244,3 +244,5 @@
 ALLOW_DISASTROUS_DATA_DROPS_NEVER_ENABLE_IN_PROD = env.bool(
     "ALLOW_DISASTROUS_DATA_DROPS_NEVER_ENABLE_IN_PROD", False
 )
+
+SHOW_ALL_WORKBASKET_ROWS = env.bool("SHOW_ALL_WORKBASKET_ROWS", False)
```
Comment (oskusalerma) [inline config/settings/base.py:248]: change as discussed offline

File: web/domains/case/models.py [hunk 3]
```diff
@@ -290,99 +291,121 @@ def get_workbasket_row(self, user: User) -> WorkbasketRow:
...
-                # TODO: implement this
-                admin_actions.append(
-                    WorkbasketAction(is_post=False, name="Authorise Documents", url="#TODO")
-                )
+                if applicant_actions:
+                    r.actions.append(applicant_actions)
 
+        # Applicant workbasket rows
+        else:
+            applicant_actions = self._get_applicant_actions(view_action, kwargs)
```
Comment (oskusalerma) [inline web/domains/case/models.py:314]: as dfidussced offline

File: web/domains/case/models.py [hunk 4]
```diff
@@ -290,99 +291,121 @@ def get_workbasket_row(self, user: User) -> WorkbasketRow:
...
+
+            if applicant_actions:
+                r.actions.append(applicant_actions)
+
+        return r
+
+    def _get_admin_actions(self, user, view_action, task, kwargs):
```
Comment (oskusalerma) [inline web/domains/case/models.py:321]: typing!

File: web/domains/case/models.py [hunk 5]
```diff
@@ -290,99 +291,121 @@ def get_workbasket_row(self, user: User) -> WorkbasketRow:
...
 
-                for fir in self.further_information_requests.open():  # type: ignore[attr-defined]
-                    applicant_actions.append(
-                        WorkbasketAction(
-                            is_post=False,
-                            name="Respond FIR",
-                            url=reverse("case:respond-fir", kwargs=kwargs | {"fir_pk": fir.pk}),
-                        )
-                    )
+    def _get_applicant_actions(self, view_action, kwargs):
```
Comment (oskusalerma) [inline web/domains/case/models.py:365]: typing!

File: web/management/commands/add_dummy_data.py [hunk 6]
```diff
@@ -95,4 +70,104 @@ def handle(self, *args, **options):
         )
         importer.offices.add(office)
 
-        self.stdout.write("Created dummy importer/exporter and associated admin user with them")
+        self.stdout.write("Created dummy importer/exporter")
+
+        self.create_user(
+            username="ilb_admin",
+            password=options["password"],
+            firstname="Bob",
```
Comment (oskusalerma) [inline web/management/commands/add_dummy_data.py:78]: can we pick a gender-neutral name for admin :) this is 2021.

ashley?

**Review by kevinetienne — APPROVED** [review]

**Review by oskusalerma — CHANGES_REQUESTED** [review]

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**